samples/semihosting/histogram/histogram.cpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+#include <cstdint>
+#include <array>
+#include <iostream>
+#include "raw_pixels.hpp"
+
+#define IMG_ROWS 100
+#define IMG_COLS 100
+
+static_assert(IMG_ROWS * IMG_COLS <= RAW_PIXELS_SIZE, "Incompatible size");
+
+int main(void)
+{
+    // Number of experiment runs
+    int no_runs = 2;
+
+    // https://docs.opencv.org/master/d3/d63/classcv_1_1Mat.html
+    cv::Mat src_new(IMG_ROWS, IMG_COLS, cv::IMREAD_GRAYSCALE, (void *)raw_pixels);
```

Comment:
`cv::IMREAD_GRAYSCALE` -- wrong cv::Mat type.

Reply:
oh - what would be the right type?

Reply:
@asmorkalov  - I need some input here to be able to address your comment.

Thank you!

Francesco

Reply:
it should be `CV_8UC1`, if it's grayscale matrix.

Reply:
Like this?

```
cv::Mat src_new(IMG_ROWS, IMG_COLS, CV_8UC1, (void *)raw_pixels);
```